recipe/meta.yaml

```diff
@@ -7,18 +7,30 @@ package:
 source:
     fn:  xz-{{ version }}.tar.bz2
     url: http://tukaani.org/xz/xz-{{ version }}.tar.gz
-    sha256: 5dcffe6a3726d23d1711a65288de2e215b4960da5092248ce63c99d50093b93a
+    sha256: 73df4d5d34f0468bd57d09f2d8af363e95ed6cc3a4a86129d2f2c366259902a2 
 
 build:
     number: 1
-    skip: True  # [win]
+    # limited to VS 2015 on Win for now - for sake of C99 support
+    msvc_compiler: 14.0
+
+requirements:
+    run:
+       # limited to VS 2015 on Win for now - for sake of C99 support
+       - vs2015_runtime
```

Comment:
As much as we dislike it, could we do this with `python` and an appropriate `skip`? Otherwise, I'm not sure our build system will behave right.